pages/docs/cloud/repository-integration.mdx

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Please add this page to the sidebar under `Cloud` so people can get to it more easily